classification.yaml

```diff
@@ -0,0 +1,791 @@
+  status: feature-complete
+  priority: 1a
+  notes:
+    This was a major architectural change from Gen2, but it's now something we do all the time.
+DMS-MWBT-REQ-0060:
+  status: feature-complete
+  priority: 1a
+  notes:
+    We haven't used this in production that I am aware of (production always uses configuration defaults), but the functionality is included in `daf_butler` unit test suite.
+DMS-MWBT-REQ-0063:
+  status: feature-complete
+  priority: 1a
+  notes:
+    Not regularly exercised in production, but functionality is tested in daf_butler unit tests.
+DMS-MWBT-REQ-0064:
+  status: feature-complete
+  priority: 1a
+  notes:
+    In regular use in production and included in unit tests.
+DMS-MWBT-REQ-0065:
+  status: feature-incomplete
+  priority: 1a
+  missing:
+    We can do remote writes to object stores, but no database datastore yet.  Note that when we do have one, it probably won't actually involve remote writes to a SQL database *during execution*, because that doesn't scale, but the data will land in a SQL database eventually, and the user shouldn't care about that distinction (so neither should the requirement).
+DMS-MWBT-REQ-0066:
+  status: feature-complete
+  priority: 1a
+  notes:
+    In regular use in production and included in unit tests.
+DMS-MWBT-REQ-0072:
+  status: feature-incomplete
+  priority: 1a
+  missing:
+    Driving use case is the subject of DMTN-203 (still needs to be written).
+DMS-MWBT-REQ-0073:
+  status: feature-complete
+  priority: 1a
+  notes:
+    Blocking writes works in all of the Datastores we provide, and we've learned from mishaps at CCIN2P3 that not blocking until writes are fully flushed very quickly causes problems with batch execution.
+DMS-MWBT-REQ-0074:
+  status: feature-complete
+  priority: 1a
+  notes:
+    Attempted-clobber being an error is in fact the only available behavior at the butler level.
+DMS-MWBT-REQ-0075:
+  status: feature-complete
+  priority: 1a
+  notes:
+    Satisfied by what `Butler.put` returns.  Exercised all over our unit tests.
+DMS-MWBT-REQ-0076:
+  status: feature-complete
+  priority: 1a
+  notes:
+    This is certainly the usual behavior, and we regard it as a bug when it is violated, and we don't currently have any known bugs of this type.
+DMS-MWBT-REQ-0077:
+  status: feature-complete
+  priority: 1a
+  notes:
+    I think we can say we can already do this with a combination of `get` and `put`.  If we can't, I'm not sure it's worth trying to do more under butler, because it's easier to do above it.
+DMS-MWBT-REQ-0078:
+  status: feature-complete
+  priority: 1a
+  notes:
+    This behavior is not guaranteed by code, but it is the way we have configured our filename templates.
+DMS-MWBT-REQ-0079:
+  status: unclear
+  priority: 1b
+  notes:
+    This isn't how we are doing batch processing, and we don't plan to start doing it this way.  Low-level functionality for doing this exists, but writing the high-level part in BPS is probably a waste of time.
```

Comment:
Should be discussed with -0052.

Reply:
Yes, if Rucio/PanDA does that automatically and we are using local-file system datastores then it all Just Works.